### PR TITLE
Script for reproducing defect #955.

### PR DESCRIPTION
**Pull request by IGapchuk**
ATF Test Scripts to check [#955](https://github.com/smartdevicelink/sdl_core/issues/955)

This PR is **ready** for review.

### Summary
A test script to verify that issue is fixed.

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by dboltovskyi** [comment]
Livio team please notice script is ready for review.

File: test_scripts/Defects/6_1/955_incorrect_response_on_send_SystemRequest.lua [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---------------------------------------------------------------------------------------------
+-- GitHub issue: https://github.com/smartdevicelink/sdl_core/issues/955
+---------------------------------------------------------------------------------------------
+-- Steps:
+-- 1. Register app
+-- 2. Activate app
+-- 3. Send SystemRequest(PROPRIETARY, "/test")
+--
+-- Expected:
+-- SDL responses with INVALID DATA result code
```
Comment (ShobhitAd) [inline test_scripts/Defects/6_1/955_incorrect_response_on_send_SystemRequest.lua:10]:
```suggestion
-- SDL responds to SystemRequest with INVALID DATA result code
```

**Comment by dboltovskyi** [comment]
@ShobhitAd Please find update in 10e1084e68ff65865a6c563194b83b8c0b9860e1

**Pull request merged**